packages/backend/src/ee/services/CommercialSlackIntegrationService.ts

```diff
@@ -26,4 +43,12 @@ export class CommercialSlackIntegrationService extends SlackIntegrationService<C
 
         return response;
     }
+
+    async deleteInstallationFromOrganizationUuid(user: SessionUser) {
+        await super.deleteInstallationFromOrganizationUuid(user);
+
+        await this.aiAgentModel.deleteSlackIntegrations({
+            organizationUuid: user.organizationUuid!,
```

Comment:
I think this is safe, but maybe let's do a check if the org uuid doesn't exist?

Reply:
went with the bang as `super.deleteInstallationFromOrganizationUuid` performs that check

Reply:
ah nice!